apps/web/src/actions/sdk/addMessagesAction.ts

```diff
@@ -26,12 +26,14 @@ export async function addMessagesAction({
   messages,
 }: AddMessagesActionProps) {
   const sdk = await createSdk().then((r) => r.unwrap())
-  const stream = createStreamableValue<ChainEvent, Error>()
+  const stream = createStreamableValue<
+    { event: StreamEventTypes; data: ChainEventDto },
+    Error
+  >()
 
-  const response = sdk.addMessages({
-    params: { documentLogUuid, messages, source: LogSources.Playground },
-    onMessage: (chainEvent) => {
-      stream.update(chainEvent)
+  const response = sdk.chat(documentLogUuid, messages, {
```

Comment:
I'm changing the public API to follow the pattern of positional argument for mandatory arguments and a last hash object for any optional argument. I think it's a better dev experience.

Reply:
I'm also changing the two main methods from runDocument and addMessages to `run` and `chat`. It's easier and faster to write at the cost of sliiiightly less expresiveness.